src/components/spotlight/spotlight.story.js

```diff
@@ -0,0 +1,217 @@
+import { withNotes } from '@ecl/storybook-addon-notes';
+import withCode from '@ecl/storybook-addon-code';
+import { getColorModeControls, correctPaths } from '@ecl/story-utils';
+import getSystem from '@ecl/builder/utils/getSystem';
+
+// Import data for demos
+import spotlightDataImage from './demo/data--image';
+import spotlight from './spotlight.html.twig';
+import notes from './README.md';
+
+const getArgs = (data) => {
+  const args = {
+    show_header: true,
+    show_anchor: true,
+    show_credit: true,
+    font_size: 'l',
+    header: data.header,
+    title: data.title,
+    credit: data.credit,
+    full_width: true,
+    sidebarContent: false,
+    gridContent: false,
+  };
+  if (data.picture) {
+    args.image = data.picture.img.src || '';
+  }
+  if (getSystem() === 'ec') {
+    args.color_mode = 'default';
+  }
+
+  return args;
+};
+
+const getArgTypes = (data) => {
+  const argTypes = {
+    ...getColorModeControls(),
+    show_header: {
+      name: 'header',
+      type: { name: 'boolean' },
+      description: 'Show the header',
+      table: {
+        category: 'Optional',
+      },
+    },
+    show_anchor: {
+      name: 'anchor',
+      type: { name: 'boolean' },
+      description: 'Show the visual anchor',
+      table: {
+        category: 'Optional',
+      },
+    },
+    show_credit: {
+      name: 'credit',
+      type: { name: 'boolean' },
+      description: 'Show the credit',
+      table: {
+        category: 'Optional',
+      },
+    },
+    font_size: {
+      name: 'font size',
+      type: 'select',
+      description: 'Change font size',
+      options: ['m', 'l'],
+      control: {
+        labels: {
+          m: 'medium',
+          l: 'large',
+        },
+      },
+      mapping: {
+        medium: 'm',
+        large: 'l',
+      },
+      table: {
+        type: 'string',
+        defaultValue: { summary: 'm' },
+        category: 'Display',
+      },
+    },
+    full_width: {
+      name: 'full width',
+      type: 'boolean',
+      description: 'Take the full width of the viewport when in a container',
+      mapping: {
+        0: false,
+        1: true,
+      },
+      table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: 'false' },
+        category: 'Display',
+      },
+    },
```

Comment:
we should make this conditional, only when sidebarContent is false, otherwise we reset the value in the data we pass but the control is still there showing "true".